File: 1. Iniciante/1101.c
#include <stdio.h>

int main(){
    int m, n, soma=0, f, i;
    scanf("%d %d", &m, &n);
    while(m > 0 && n > 0){
        if(n>m){
            f = m;
            m = n;
            n = f;
        }
        for(i=n;i<=m;i++){
            printf("%d ", i);
            soma+=i;
        }
        printf("Sum=%d\n", soma);
        soma=0;
        scanf("%d %d", &m, &n);
    }
    return 0;
}

File: 1. Iniciante/1101.py
from sys import stdout
M, N = map(int, sorted(input().split()))

while M > 0:
    soma = 0
    for i in range(M,N+1):
        soma+= i
        stdout.write(f'{i} ')
    print(f"Sum={soma}")
    M, N = map(int, sorted(input().split()))